statemachine/contrib/diagram.py

```diff
@@ -137,15 +140,16 @@ def import_sm(qualname):
     module = importlib.import_module(module_name)
     smclass = getattr(module, class_name, None)
     if not smclass or not issubclass(smclass, StateMachine):
-        raise ValueError("{} is not a subclass of StateMachine".format(class_name))
+        raise ValueError(
+            "{} is not a subclass of StateMachine".format(class_name))
```

Comment:
nitpick: These changes should be reverted to the library style (`line-length = 99`). If you run the configured linters they'll be automatically reverted.